public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -127,6 +128,7 @@ export class ElasticDatasource
   includeFrozen: boolean;
   isProxyAccess: boolean;
   databaseVersion: SemVer | null;
+  defaultQueryMode?: QueryType;
```

Comment:
This doesn't seem to be used anywhere. Can we remove it?

Reply:
It is used here `components/QueryEditor/ElasticsearchQueryContext.tsx`